portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.testcase

```diff
@@ -0,0 +1,163 @@
+        
+        Pause(locator1 = "3000");
+
+        Click(locator1 = "//span[contains(text(),'Control Panel')]");
+
+        Pause(locator1 = "3000");
+
+        AssertElementPresent(locator1 ="//h2[@id='nav_control_panel.configuration']");
+
+        Pause(locator1 = "3000");
+    } 
+
+   /* @description = "Set Screen size to custom and validate values"
+    @priority = "5"
+    test SetScreenSizeAndValidateValues {
+    }*/
+
+    @description = "Access Portal Global Menu and validate changes in SIMULATION with alterations in SCREEN SIZE and validate after all"
+    @priority = "5"
+    test CreatingNewForm {
+        task("Going to Forms") {
+            FormsAdmin.addForm();
+    }
+        task("Editing Title") {
+            Form.editName(formName = "Liferay's Forms");
+    }
+        task("Adding and editing Text Field") {
+            Form.addTextField();
+   
+            FormViewBuilder.editFieldLabel(
+			fieldLabel = "Text",
+			label = "Full Name");
+
+            Form.closeSidebar();
+ 
+    }
+        task("Adding and editing Numeric Field") {
+            Form.gotoAddField(
+			fieldPositionNumber = 1,
+			fieldType = "Numeric");
+
+            Form.editFieldLabelMetal(fieldValue= "Age");
+            
+    }
+        task("Translate to Portuguese") {
+            Form.addTranslation(localizedLanguage= "Portuguese (Brazil)");
+            Form.save();
+    }
+        task("Editing TITLE, FULL NAME and AGE") {
+            Form.selectTranslation(localizedLanguage= "Portuguese (Brazil)");
+        
+            Form.editName(formName = "Forms da Liferay");
+
+            FormViewBuilder.editFieldLabel(
+            fieldLabel = "Full Name",
+            label = "Nome Completo");
+           
+            FormViewBuilder.editFieldLabel(
+            fieldLabel = "Age",
+            label = "Idade");
+
+            Form.save();
+            Form.publishForm();
+    }
+        task("Going to Published Form") {
+            FormsAdminNavigator.gotoPublishedForm();
+    }
+        task("New information on published forms") {
+            FormViewBuilder.editText(
+            fieldName = "Full Name",
+            fieldValue = "Daniel");
+
+            FormViewBuilder.editText(
+            fieldName = "Age",
+            fieldValue = "17");
+
+            FormPortlet.clickSubmit();
+
+		    Form.submitAgain();
+
+    }
+        task("Changing language") {
+            FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
+            FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "português-Brasil");
+    }
+        task("New information on published forms") {
+            FormViewBuilder.editText(
+            fieldName = "Nome Completo",
+            fieldValue = "Danielzinho");
+
+            FormViewBuilder.editText(
+            fieldName = "Idade",
+            fieldValue = "18");
+
+            FormPortlet.clickSubmit();
+
+		    Form.submitAgain();
```

Comment:
Why are there empty spaces here? For these and other fixes, you must do the following:

- After making the commits, go to the "dev/projects/liferay-portal/portal-impl" folder and execute the following command: ant format-source-current-branch. You can go to the ".bashrc" folder and check if there is an alias for it.

- Wait for the command to run. It may last for a few minutes.

- Check the corrections that were made and commit them. For example: "LPS 0000 SF